pages/login.tsx

```diff
@@ -6,12 +6,23 @@ import {
     Input,
     FormControl,
 } from "@chakra-ui/react";
+import { useState } from "react";
+import useLocalStorage from "@utils/useLocalStorage";
+import { signIn } from "next-auth/client";
 
 /**
  * This is the page that a user will use to either login or register
  * to the SDC platform
  */
 export default function Login(): JSX.Element {
+    // hook for storing the email
+    const [email, setEmail] = useState("");
+    // save the email into localstorage for email verification page
+    const [, setLocalStorageEmail] = useLocalStorage(
+        "sdc-email-verification",
+        "",
+    );
+
     return (
         <Center h="500px">
```

Comment:
Once https://github.com/uwblueprint/social-diversity-for-children/pull/51 is merged, can you merge staging onto your branch and wrap this page in the `SDCWrapper` component? That'll add the navbar and footer to the page. The PR is small and should be merged sometime today

Reply:
Update: it has been merged